tests/ext/commands/test_view.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: MIT
+
+
+import pytest
+
+from disnake.ext.commands.errors import (
+    ExpectedClosingQuoteError,
+    InvalidEndOfQuotedStringError,
+    UnexpectedQuoteError,
+)
+from disnake.ext.commands.view import StringView
+
+
+class TestView:
+    @pytest.mark.parametrize(
+        ("text", "expected_word"),
+        [
+            ("hello", "hello"),
+            ("how are you", "how"),
+            ("  bugs are fun", ""),
+            ('"""', '"""'),
+            ("''", "''"),
+            ('hone"stl"y, quotes.', 'hone"stl"y,'),
+        ],
+    )
+    def test_get_word(self, text, expected_word):
+
+        view = StringView(text)
+
+        word = view.get_word()
+
+        assert word == expected_word
+
+    @pytest.mark.parametrize(
+        ("text", "expected_word"),
+        [
+            ("hello", "hello"),
+            ("how are you", "how"),
```

Comment:
```suggestion
            ("how are you", "how"),
            ('"some quotes" here', "some quotes"),
```
The other test cases didn't really test the core functionality, this one would be the most basic case.